app/experimenter/static/rapid/__tests__/experimentDetails.test.tsx

```diff
@@ -285,6 +285,32 @@ describe("<ExperimentDetails />", () => {
     });
   });
 
+  it("Initializes the svelte code as expected", async () => {
```

Comment:
Is this just checking that it doens't throw any errors?  Is there any way to assert that the svelte parts are actually rendered?

Reply:
I guess right now it's not dumping out any content so there's no content to look for, since it's behind a feature flag maybe you can just put a little piece of text or something and assert that that's present?

Reply:
nothing gets rendered in this PR, but yea probably in future PRs we can do that.

Right now, this is just to get code coverage to 100%

Reply:
Oh yeah true coverage helps here